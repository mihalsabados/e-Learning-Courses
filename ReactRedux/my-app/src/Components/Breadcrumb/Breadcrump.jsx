import React  from 'react';
import './Breadcrump.css';

export default function Breadcrumb() {
    return (
        <div className="breadcrumb-container">
            <div className="breadcrumb-holder">
                <p>Home Page &nbsp; {'>'} &nbsp; e-Learning Courses</p>
            </div>
            <div className="user-role">
                <p>admin</p>
            </div>
        </div>
    );
  }