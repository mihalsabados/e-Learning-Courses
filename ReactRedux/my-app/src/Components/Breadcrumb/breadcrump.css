
.breadcrumb-container{
    display: flex;
    margin-top: 30px;
    height: 30px;
}

.breadcrumb-holder{
    background-color: white;
    width: 80%;
    color: rgb(63, 63, 63);
    padding-left: 15px;
}

.breadcrumb-holder p{
    margin-top: 7px;
    color: #706f8e;
}

.user-role{
    width: 20%;
    background-color: #676767;
    padding-left: 15px;
}

.user-role p{
    color: #d9d3cb;
    margin-top: 7px;
}