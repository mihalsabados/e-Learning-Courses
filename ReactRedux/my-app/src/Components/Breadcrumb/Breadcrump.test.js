/* eslint-disable no-undef */
import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import Breadcrumb from "./Breadcrump";

test("Breadcrumb component should render", async () => {
	render(<Breadcrumb />);

	// Assert that the breadcrumb text is present
	expect(
		screen.getByText("Home Page > e-Learning Courses")
	).toBeInTheDocument();

	// Assert that the user role is present
	expect(screen.getByText("admin")).toBeInTheDocument();
});

test("Breadcrumb component snapshot", () => {
	const component = renderer.create(<Breadcrumb />);
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
