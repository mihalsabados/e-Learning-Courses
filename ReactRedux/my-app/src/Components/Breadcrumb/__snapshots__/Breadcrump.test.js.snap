// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb component snapshot 1`] = `
<div
  className="breadcrumb-container"
>
  <div
    className="breadcrumb-holder"
  >
    <p>
      Home Page   
      &gt;
         e-Learning Courses
    </p>
  </div>
  <div
    className="user-role"
  >
    <p>
      admin
    </p>
  </div>
</div>
`;
